usaspending_api/download/filestreaming/download_generation.py

```diff
@@ -553,3 +559,24 @@ def _kill_spawned_processes(download_job=None):
             spawn_of_job.kill()
         except ps.NoSuchProcess:
             pass
+
+
+def create_empty_data_file(
+    source: DownloadSource,
+    download_job: DownloadJob,
+    working_dir: str,
+    piid: str,
+    assistance_id: str,
+    zip_file_path: str,
+    file_format: str,
+) -> None:
+    data_file_name = build_data_file_name(source, download_job, piid, assistance_id)
+    extension = FILE_FORMATS[file_format]["extension"]
+    source.file_name = f"{data_file_name}.{extension}"
+    source_path = os.path.join(working_dir, source.file_name)
+    write_to_log(
+        message=f"Skipping download of {source.file_name} due to no valid columns provided", download_job=download_job
+    )
+    with open(source_path, "w") as _:
```

Comment:
Nothing wrong with opening a file for a nop. However, every change Kirk and I get we preach the wonders of [Pathlib](https://docs.python.org/3/library/pathlib.html)
See Pathlib's [touch()](https://docs.python.org/3/library/pathlib.html#pathlib.Path.touch)